WcaOnRails/app/webpacker/lib/show-schedule.js

```diff
@@ -1,5 +1,6 @@
 import _ from 'lodash';
 import { events } from './wca-data.js.erb';
+import { getTextColor } from './utils/calendar';
```

Comment:
Don't think this is used?

Reply:
It actually is used: The tab that statically shows a competition's schedule in the public view currently relies on this file.
We can only get rid of it once that other schedule has also been migrated (in a follow-up PR)